db/init.go

```diff
@@ -104,12 +107,35 @@ func Migrate() error {
 	}
 	defer db.Close()
 
-	if err := goose.Up(db, "migrations", goose.WithAllowMissing()); err != nil {
-		return err
+	for {
+		err = goose.UpByOne(db, "migrations", goose.WithAllowMissing())
+		if err == goose.ErrNoNextVersion {
+			break
+		}
+		if err != nil {
+			return err
+		}
 	}
 	return nil
 }
 
 func GetDB() (*sql.DB, error) {
 	return sql.Open("pgx", pgxConnectionString)
 }
+
+func ConvertNamedParams(sql string, namedArgs map[string]interface{}) (string, []interface{}) {
```

Comment:
Also, lets use the variable `query` here

Reply:
Same function is removed in favour of chained query